ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/Clusters/ClusterDetails.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback } from 'react';
+import { useParams } from 'react-router-dom';
+import {
+    Alert,
+    Breadcrumb,
+    BreadcrumbItem,
+    Divider,
+    PageSection,
+    Skeleton,
+} from '@patternfly/react-core';
+
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import PageTitle from 'Components/PageTitle';
+import useRestQuery from 'hooks/useRestQuery';
+import { complianceEnhancedCoveragePath } from 'routePaths';
+import { getSingleClusterCombinedStats } from 'services/ComplianceEnhancedService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+import ClusterDetailsHeader from './ClusterDetailsHeader';
+
+function ClusterDetails() {
+    const { clusterId } = useParams();
+
+    const listQuery = useCallback(() => getSingleClusterCombinedStats(clusterId), [clusterId]);
+    const { data: clusterStats, loading: isLoadingClusterInfo, error } = useRestQuery(listQuery);
+
+    const renderClusterNameBreadcrumb = () => {
+        if (error) {
+            return null;
+        }
+
+        return (
+            <BreadcrumbItem isActive>
+                {isLoadingClusterInfo ? (
+                    <Skeleton screenreaderText="Loading cluster name" width="150px" />
+                ) : (
+                    clusterStats?.cluster.clusterName
+                )}
+            </BreadcrumbItem>
+        );
+    };
+
+    return (
+        <>
+            <PageTitle title="Compliance Cluster Details" />
+            <PageSection variant="light" className="pf-u-py-md">
+                <Breadcrumb>
+                    <BreadcrumbItemLink to={`${complianceEnhancedCoveragePath}?tableView=Clusters`}>
+                        Clusters
+                    </BreadcrumbItemLink>
+                    {renderClusterNameBreadcrumb()}
+                </Breadcrumb>
+            </PageSection>
+            <Divider component="div" />
+            {error || clusterStats === null ? (
+                <Alert variant="warning" title="Unable to cluster details" component="div" isInline>
+                    {getAxiosErrorMessage(error)}
+                </Alert>
+            ) : (
+                <>
+                    <PageSection variant="light">
```

Comment:
For what it is worth, I have started adding `component="div"` for `PageSection` elements whose main purpose is layout instead of page hierarchy, so with inspector you would see `div` and `div` for presentation, followed by a more semantic `section` element which contains the table.

Reply:
I hadn't even noticed that as an option, definitely something I should be using.